codes/classical/properties/covering.yml

```diff
@@ -0,0 +1,46 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: covering
+
+name: 'Covering code'
+
```

Comment:
introduced: \cite{ref}?

Reply:
is there really an appropriate reference for this? they are simply a property and most of these codes have some rho radius. I didn't see any reference mentioning first people to realize covering property?

Reply:
I agree: there does not seem to be a good reference for this; I suggest to leave "introduced in" empty. Moreover, covering code is a property more than a class of codes. Not even most, but every code has a covering radius; therefore I am for deleting this from the database altogether.